src/components/container/SurveyTaskList/SurveyTaskList.stories.tsx

```diff
@@ -10,20 +10,82 @@ export default {
 	}
 };
 
-const render = (args: SurveyTaskListProps) => {
-    return <Layout colorScheme="auto">
-        <SurveyTaskList {...args} />
-    </Layout>;
+// Ease up typing for the 'flattened' sortBehavior props
+const render = (args: any) => {
+	const { 'sortBehavior.type': type, 'sortBehavior.direction': direction, 'sortBehavior.userDefinedOrder': userDefinedOrder, ...rest } = args;
+	const completeArgs: SurveyTaskListProps = {
+		...rest,
+		sortBehavior: {
+			type: type,
+			direction: direction,
+			userDefinedOrder: userDefinedOrder
+		}
+	};
+
+	return (
+		<Layout colorScheme="auto">
+			<SurveyTaskList {...completeArgs} />
+		</Layout>
+	);
 };
 
 export const Incomplete = {
 	args: {
 		limit: 3,
 		status: 'incomplete',
 		title: 'Incomplete Tasks',
-		previewState: "IncompleteTasks",
-		variant: "singleCard",
-		sequential: false
+		previewState: 'IncompleteTasks',
+		variant: 'singleCard',
+		sequential: false,
+		'sortBehavior.type': 'dueDate',
+		'sortBehavior.direction': 'ascending'
+	},
+	argTypes: {
+		sortBehavior: {
```

Comment:
We have a helper function for this.  You can add this to the `argTypes` with a list of any properties you'd like to have hidden.

```
...argTypesToHide(['sortBehavior'])
```